examples/evm/buildExportTx-pchain.ts

```diff
@@ -0,0 +1,86 @@
+import { Avalanche, BN } from "../../src"
+import {
+  PlatformVMAPI,
+  KeyChain as PlatformKeyChain
+} from "../../src/apis/platformvm"
+import {
+  EVMAPI,
+  KeyChain as EVMKeyChain,
+  UnsignedTx,
+  Tx
+} from "../../src/apis/evm"
+import {
+  PrivateKeyPrefix,
+  DefaultLocalGenesisPrivateKey,
+  Defaults,
+  costExportTx,
+  ONEAVAX
+} from "../../src/utils"
+
+const ip: string = "localhost"
+const port: number = 9650
+const protocol: string = "http"
+const networkID: number = 12345
+const avalanche: Avalanche = new Avalanche(ip, port, protocol, networkID)
+const pchain: PlatformVMAPI = avalanche.PChain()
+const cchain: EVMAPI = avalanche.CChain()
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`
+const pKeychain: PlatformKeyChain = pchain.keyChain()
+const cKeychain: EVMKeyChain = cchain.keyChain()
+pKeychain.importKey(privKey)
+cKeychain.importKey(privKey)
+const pAddressStrings: string[] = pchain.keyChain().getAddressStrings()
+const cAddressStrings: string[] = cchain.keyChain().getAddressStrings()
+const pChainBlockchainIdStr: string = Defaults.network[networkID].P.blockchainID
+const avaxAssetID: string = Defaults.network[networkID].X.avaxAssetID
+const cHexAddress: string = "0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
+const Web3 = require("web3")
+const path: string = "/ext/bc/C/rpc"
+const web3 = new Web3(`${protocol}://${ip}:${port}${path}`)
+const threshold: number = 1
+
+const main = async (): Promise<any> => {
+  let balance: BN = await web3.eth.getBalance(cHexAddress)
+  balance = new BN(balance.toString().substring(0, 17))
+  const baseFeeResponse: string = await cchain.getBaseFee()
+  const baseFee = new BN(parseInt(baseFeeResponse, 16))
+  const txcount = await web3.eth.getTransactionCount(cHexAddress)
+  const nonce: number = txcount
+  const locktime: BN = new BN(0)
+  let avaxAmount: BN = balance.sub(baseFee)
+  let fee: BN = baseFee
+
+  let unsignedTx: UnsignedTx = await cchain.buildExportTx(
+    avaxAmount,
+    avaxAssetID,
+    pChainBlockchainIdStr,
+    cHexAddress,
+    cAddressStrings[0],
+    pAddressStrings,
+    nonce,
+    locktime,
+    threshold,
+    fee
+  )
+  const exportCost: number = costExportTx(unsignedTx)
+  avaxAmount = balance.sub(baseFee.mul(new BN(exportCost))).sub(ONEAVAX)
+  fee = baseFee.mul(new BN(exportCost))
```

Comment:
Can we put `fee` on the line before `avaxAmount` so that it can be used when calculating `avaxAmount` instead of doing this multiplication twice?

Reply:
Same question with subtracting one avax as above

Reply:
> Can we put fee on the line before avaxAmount so that it can be used when calculating avaxAmount instead of doing this multiplication twice?

Resolved in [c3d207ead88fbef754687ce15042ebd05601cb01](https://github.com/ava-labs/avalanchejs/commit/c3d207ead88fbef754687ce15042ebd05601cb01

> Same question with subtracting one avax as above

Same answer as above